src/seismometer/html/resources/info.html

```diff
@@ -3,32 +3,33 @@
         text-align: left !important;
     }
 </style>
-
-<h3>Summary</h3>
-The preloaded data covers {{ num_predictions }} predictions over {{ num_entities }} entities from the dates {{ start_date }} to {{ end_date }}.
-<table>
-    <thead>
+<div style="width: 100%; max-width: 1400px;">
```

Comment:
Is there a way to parametrize this `"width: 100%; max-width: 1400px;"`? or make it some kind of constant?

If it's more burdensome/obfuscating than its worth then it is fine, but have to ask

Reply:
We could pass the style in as a parameter, and then hard code it on the load template function. that's not too bad, and unit tests would catch it by default. 

```html
<div style="{{ display_style }}">
```

The second option is to display an HTML element earlier that includes a CSS class in a style tag so that we can just reference that style. This would be ideal, but I'm not sure what the best hook is to return the css since we need that to be displayed in the notebook to get the CSS, generating the HTML element is not enough. 

```python
def _set_ux_css()
    from IPython.display import HTML
    return HTML(Path('custom.css').read_text())
```

The second option would be better, since it gives us a path to add more CSS content in the future, but since I can't think of a good way to load an extra CSS file (other than requiring themes/extensions) I'm going to stick with the first option.